actionview/lib/action_view/helpers/translation_helper.rb

```diff
@@ -111,7 +111,7 @@ def translate(key, **options)
 
       # Delegates to <tt>I18n.localize</tt> with no additional functionality.
       #
-      # See https://www.rubydoc.info/github/svenfuchs/i18n/master/I18n/Backend/Base:localize
+      # See https://www.rubydoc.info/github/ruby-i18n/i18n/master/I18n%2FBackend%2FBase:localize
```

Comment:
Where possible, I think it would be good to link to the "gem" version of the documentation.  For example:

```suggestion
      # See https://www.rubydoc.info/gems/i18n/I18n/Backend/Base:localize
```

Reply:
Sounds good, let me check the others!.